etc/spack/defaults/packages.yaml

```diff
@@ -14,10 +14,16 @@
 #   ~/.spack/packages.yaml
 # -------------------------------------------------------------------------
 packages:
+  ibm-mpi:
```

Comment:
The problem is that not every Spack user has ibm-mpi 10.1.0.2 installed in /opt/ibm/spectrum_mpi.